subprojects/dependency-management/src/integTest/groovy/org/gradle/integtests/resolve/ProjectDependencyResolveIntegrationTest.groovy

```diff
@@ -217,7 +217,7 @@ project(':b') {
             configurations { compile }
             dependencies { compile project(path: ':a', configuration: 'compile'), project(path: ':b', configuration: 'compile') }
             task test(dependsOn: configurations.compile) << {
-                assert configurations.compile.collect { it.name } == ['a.jar', 'b-late.jar']
+                assert configurations.compile.collect { it.name } == ['a.jar', 'b-early.jar']
```

Comment:
Previously it was possible to change the version number of a project after the task graph was built, but now it's not possible -- hence the change here.